### DISCUSSION TITLE
图床暂时停摆

### DISCUSSION THREAD

**Discussion by z0z0r4** [LTCatt/PCL2Help#371] [General]
`All your data is backed up. You must pay 0.0059 BTC to 1GbLiucJ7fhsM3sYrPKHvZ5mUW2p4AYW7p In 48 hours, your data will be publicly disclosed and deleted. (more information: go to http://iplis.ru/data01)After paying send mail to us: dzen+1jqfmv@onionmail.org and we will provide a link for you to download your data.`

最近开了个沟槽，没来得及关防火墙，给锁了

正在重装、恢复备份中，暂时停摆

**Comment by z0z0r4** [comment]
~~停摆是顺便的，一直想重装没借口.jpg~~

**Comment by WorldHim** [comment]
草原来是你的数据库

**Reply by allMagicNB** [reply]
你才知道啊（

**Reply by WorldHim** [reply]
> 你才知道啊（

openbmclapi 群里看到的，没注意是谁

**Comment by z0z0r4** [comment]
emm，修完了
